content/source/docs/cloud/api/changelog.html.md

```diff
@@ -0,0 +1,58 @@
+---
+page_title: "API changelog - API Docs - Terraform Cloud and Terraform Enterprise"
+layout: "cloud"
+page_id: "api-changelog"
+---
+
+[breaking]: ./api/changelog/breaking.png "Breaking"
+[cloud]: ./api/changelog/cloud.png "Cloud"
+[enterprise]: ./api/changelog/enterprise.png "Enterprise"
+
+# API Changelog
+
+Keep track of changes to the Terraform Cloud and Terraform Enterprise APIs.
+
+### 2021-01-21
+
+* ![cloud][] A new Invoices API is available.
+* The terraform-version attribute in the Workspaces API is now deprecated. Please use terraform-requirement instead.
+* ![enterprise][] A new Module Sharing API is available for sharing Terraform modules across organizations.
+* ![breaking][] The X API will no longer return Y. To do Z, use this workaround. This is a breaking change necessary for the security of the platform, covered in our API stability policy.
+
+### 2021-05-27
+
+* Added the [delete endpoint](https://www.terraform.io/docs/cloud/api/agents.html#delete-an-agent) to the [Agents API](https://www.terraform.io/docs/cloud/api/agents.html).
+
+### 2021-04-28
+
+* Added the `refresh`, `refresh-only`, and `replace-addrs` attributes to the [Runs API](https://www.terraform.io/docs/cloud/api/runs.html).
+
+### 2021-04-16
+
+* Introduce [Controlled Remote State Access](https://www.hashicorp.com/blog/announcing-controlled-remote-state-access-for-terraform-cloud-and-enterprise).
+  * Updated [Admin Settings API](https://www.terraform.io/docs/cloud/api/admin/settings.html)
+  * Updated [Workspaces API](https://www.terraform.io/docs/cloud/api/workspaces.html).
+  * Added [Remote State Consumers](https://www.terraform.io/docs/cloud/api/workspaces.html#get-remote-state-consumers).
+
+### 2021-04-13
+
+* Added a `manage-policy-overrides` property to the `organization-access` attribute in the [Teams API](https://www.terraform.io/docs/cloud/api/teams.html).
+
+### 2021-03-23
+
+* ![enterprise][] Introduce [Share Modules Across Organizations with Terraform Enterprise](https://www.hashicorp.com/blog/share-modules-across-organizations-terraform-enterprise).
```

Comment:
@bethanyr @jbonhag I referenced the blog here (broader discussion required) and only added the broader APIs being updated. Thoughts on us being more specific here?

Reply:
I have updated and added the following:
![image](https://user-images.githubusercontent.com/4753589/120729036-0b138f80-c493-11eb-88a2-5e7b3b27f343.png)